web/text_layer_builder.js

```diff
@@ -215,11 +219,16 @@ class TextLayerBuilder {
     });
 
     div.addEventListener("copy", event => {
-      const selection = document.getSelection();
-      event.clipboardData.setData(
-        "text/plain",
-        removeNullCharacters(normalizeUnicode(selection.toString()))
-      );
+      if (this.#enablePermissions) {
+        // Prevent copying.
+        event.clipboardData.setData("text/plain", "");
```

Comment:
Why are you overwriting clipboard content with an empty string ?
I checked in Acrobat and Chrome and the clipboard isn't changed.

Reply:
No good reason, so that's been fixed now :-)